samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -703,21 +725,13 @@ function main {
 
   if ! create_cos_bucket "${COS_BUCKET_NAME}" "${COS_BUCKET_REGION}" "${COS_RESOURCE_GROUP}" ; then exit 1 ; fi
 
-  # Downloading rhel locally, else the url may expire by the time the VM comes up
-  if ! [ -z "${RHEL_URL}" ];then
-    SECONDS=0
-    MSG="Downloading rhel qcow2 image completed in"
-    download_image "rhel"
-    time_taken "$MSG"
```

Comment:
could you please confirm this again from the RH doc. Please provide the supporting link here

Reply:
I cloud see the below message when i tried opening an expired download link also tested after 2hrs its working fine..(expired [link](https://access.cdn.redhat.com/content/origin/files/sha256/2a/2aa8e77a043f4731d2c30ce77f9eb3cbf2fa00a190d6f33df049a8869d3e265a/rhel-8.8-ppc64le-kvm.qcow2?user=dd127b1db16c393937fe8290dfa1e200&_auth_=1689766265_6cd1303d67151d78a5271bf684b63308'))
```
Sorry, it looks like that link has expired.
Download links are dynamic and available for 240 minutes before they expire.

If you’ve followed a bookmark, you will need to browse to the content you want to download and try again.

If you believe you have reached this page in error, contact [Red Hat Customer Service.](https://access.redhat.com/support/contact/customerService)
```